spec/support/match_contents_matcher.rb

```diff
@@ -0,0 +1,6 @@
+RSpec::Matchers.define :match_contents do |regexp|
+  match do |filename|
+    contents = IO.read(File.join(project_path, filename))
```

Comment:
Would [`Pathname(project_path, filename).read`](https://ruby-doc.org/stdlib-2.1.3/libdoc/pathname/rdoc/Pathname.html#method-i-read) yield the same value for `contents`?

Reply:
Ignore that, `Pathname` and `Pathname.new` don't take variable arguments, they both expect a single argument string.

Reply:
I couldn't find a `Pathname` constructor that takes multiple arguments, unfortunately -- so it'd still be `Pathname(File.join(project_path, filename)).read`. Dunno if that reads any more nicely.

Reply:
Ah you just said that. Yup!